test/e2e/components/vl-http-error-message.js

```diff
@@ -0,0 +1,15 @@
+const { VlElement } = require('vl-ui-core');
+const { By } = require('selenium-webdriver');
+
+class VlHttpErrorMessage extends VlElement {
+    async clickOnButton() {
+        const button = await this._getActionButton();
+        return button.click();
+    }
+
+    async _getActionButton() {
+        return this.shadowRoot.findElement(By.css('button'));
+    }
```

Comment:
Het page object kan beter generieker zijn. Aangezien we niet weten hoeveel buttons de afnemer in de error message gaat steken, kan het page object best een functie voorzien om op button x (volgorde) te klikken. Al dan niet later door @cambiph te voorzien.

Reply:
Klopt, ik ging er for now van uit van 1 knop.

Reply:
@cambiph is er nu al aan bezig